e2e/timeout-file-not-cached/src/BetterStandardPrinter.php

```diff
@@ -0,0 +1,625 @@
+<?php
```

Comment:
file should be renamed to: `SomeFixturePrinter.php`

Reply:
🤦🏼